lib/msf/core/modules/external/templates/dos.erb

```diff
@@ -13,6 +13,7 @@ class MetasploitModule < Msf::Auxiliary
           <%= meta[:references] %>
         ],
       'DisclosureDate' => <%= meta[:date] %>,
+      'Notes' => <%= meta[:notes] %>,
```

Comment:
These still need to be moved into `lib/msf/core/modules/external/templates/common_metadata.erb`